.github/workflows/wda-package.yml

```diff
@@ -78,10 +78,15 @@ jobs:
         with:
           xcode-version: "${{ env.XCODE_VERSION }}"
       - name: Create a zip of WebDriverAgentRunner${{ matrix.target }} for simulator for ${{ matrix.arch }}
-        run: sh $GITHUB_WORKSPACE/Scripts/ci/build-sim.sh
+        run: |
+          if [[ $TARGET == '_tvOS' ]]; then
+            DESTINATION=$DESTINATION_SIM_TVOS sh $GITHUB_WORKSPACE/Scripts/ci/build-sim.sh
```

Comment:
so if let say `matrix.target == iOS` we could build the variable name dynamically.
See https://github.com/orgs/community/discussions/25768 and https://stackoverflow.com/questions/16553089/dynamic-variable-names-in-bash